internal/core/commands/create_in.go

```diff
@@ -42,6 +44,7 @@ func (ch CreateIntegrationCommandHandler) Handle(ctx context.Context, query medi
 	i.Vendor = command.Vendor
 	i.Type = command.Type
 	i.Style = command.Style
+	i.TokenID = null.IntFrom(command.TokenID)
```

Comment:
I would make one exception here: if the input is `0` then leave this as null. If this seems like too much of a headache then just drop the nullable, drop the unique, make the default 0.